README.md

```diff
@@ -57,58 +54,76 @@ snap connect wifi-connect:network-setup-control core:network-setup-control
 
 Note: wifi-ap and network-manager interfaces auto-connect.
```

Comment:
I think network-setup-control is not being used anymore. Remove it from README.md